tf/mod_source_parse.go

```diff
@@ -31,6 +31,9 @@ type Source struct {
 	// Eg. github.com/mineiros-io/example
 	Path string
 
+	// PathScheme is the scheme of the path part.
+	PathScheme string
```

Comment:
Isn't it the scheme of the URL itself ? (and then it would be just Scheme maybe) :thinking: